test/functional/p2p_addrfetch.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
Consider to backport bitcoin/bitcoin#22568 in a follow-up PR. There are no any important/critical fix, just tests improvements, so, that's an out-of-scope of current PR